src/sub-pages/_single.html

```diff
@@ -55,13 +55,13 @@
         </section>
         {% endif %}
 
-        {% set content = { 'markup' : sub_page.body_content } %}
+        {% set content = { 'markup' : sub_page.body_content|safe } %}
```

Comment:
Why do we need to use the `safe` filter here if we are using it on https://github.com/cfpb/cfgov-refresh/pull/979/files#diff-8ddc445ee34233a3d2aac7be287afadeL70?